app/enterprise/2.1.x/plugins/grpc.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Introduction to Kong gRPC Plugins
+---
+
+Before we dive into the specifics of how to configure Kong's gRPC plugins, let's
+first discuss the advantages of the protocol. Unlike JSON, [gRPC](https://en.wikipedia.org/wiki/GRPC)
```

Comment:
This first sentence sounds more like a blog post than doc. Can you rephrase to make it sound less casual?

Reply:
Yeah it is bloggy. Will rework.